testsuite/tests/glbc/test_ingress_reconciliation.py

```diff
@@ -29,18 +27,15 @@ def backend_ingress(request, backend, blame):
 @backoff.on_exception(backoff.fibo, exception=httpx.ConnectError, max_time=600)
 def test_ingress_host_add(backend_ingress):
     """Creates ingress for a backend and checks for host field filled by glbc, checks host points to backend"""
-    rules = backend_ingress.rules
-    assert len(rules) == 1
-
-    backend_ingress.wait_for_hosts()
-    host = rules[0].get("host")
+    backend_ingress.wait_for_ready()
```

Comment:
also if this should not be `wait()` why shouldnt it remain `wait_for_hosts()`? At the end, is `host` without waiting needed use case? (`host` is property now, can do much more than just return value)

Reply:
I dont like mixing value access and waiting because it breaks use cases like "I have one ingress that I know is ready, and I want to access it in two different tests" so I would rather keep it separate

Reply:
then maybe `def get_host(self, wait=False):` (together with `host` property that could or could not wait by default) sounds also good